tests/test_main.py

```diff
@@ -649,3 +649,38 @@ class Config:
     m = MyModel(extra_key='extra')
     assert m.dict() == {'a': None, 'extra_key': 'extra'}
     assert m.dict(by_alias=True) == {'alias_a': None, 'extra_key': 'extra'}
+
+
+def test_root():
+    class MyModel(BaseModel):
+        __root__: str
+
+    m = MyModel(__root__='a')
```

Comment:
please test with `parse_obj` too.

Reply:
I had made tests for ``parse_obj`` in ``tests/tset_parse.py``
Would you please check it?
https://github.com/samuelcolvin/pydantic/pull/628/files#diff-7b34c8bd665994bbdcf056c03622d4a8R45